packages/breadboard-ui/src/elements/input/input.ts

```diff
@@ -295,8 +305,23 @@ export class Input extends LitElement {
       } else {
         const input = form[key];
         if (input && input.value) {
-          const parsedValue = parseValue(property.type, input);
-          data[key] = parsedValue;
+          try {
+            const parsedValue = parseValue(property.type, input);
+            data[key] = parsedValue;
+            //throw new Error("Error when parsing input values.");
+          } catch (error) {
+            if (error instanceof Error) {
+              const event = new CustomEvent(
```

Comment:
We [subclass `Event`](https://github.com/breadboard-ai/breadboard/blob/main/packages/breadboard-ui/src/events/events.ts) rather than using `CustomEvent`, so I'd like to stick to that.

Reply:
CustomEvent was changed to InputErrorEvent (see https://github.com/breadboard-ai/breadboard/pull/965#discussion_r1515953212)